library/src/main/scala/scala/scalajs/js/PropertyDescriptor.scala

```diff
@@ -10,12 +10,21 @@
 
 package scala.scalajs.js
 
-@native
+import scala.scalajs.js
```

Comment:
We've slowly been adopting this style in new code, yes. At some point we'll need a pass over the entire package to do it consistently, indeed.